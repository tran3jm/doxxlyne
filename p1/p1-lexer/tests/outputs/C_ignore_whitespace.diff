--- outputs/C_ignore_whitespace.txt	2022-09-09 04:03:50.014038634 -0400
+++ expected/C_ignore_whitespace.txt	2022-09-09 03:46:45.872089981 -0400
@@ -1,5 +1,4 @@
-DECLIT   [line 001]  39
-DECLIT   [line 002]  skdnbsljfd
-asd slj 203
-DECLIT   [line 003]  c
-0x348
+SYM   [line 001]  !
+SYM   [line 002]  ?
+SYM   [line 003]  -
+SYM   [line 004]  ,
\ No newline at end of file
