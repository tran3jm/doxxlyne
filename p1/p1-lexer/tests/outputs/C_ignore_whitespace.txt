DECLIT   [line 001]  39
DECLIT   [line 002]  skdnbsljfd
asd slj 203
DECLIT   [line 003]  c
0x348
