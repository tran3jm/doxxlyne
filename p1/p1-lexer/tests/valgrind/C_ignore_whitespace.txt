==3456366== Memcheck, a memory error detector
==3456366== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3456366== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3456366== Command: ../decaf inputs/ignore_whitespace.decaf
==3456366== 
DECLIT   [line 001]  39
DECLIT   [line 002]  skdnbsljfd
asd slj 203
DECLIT   [line 003]  c
0x348
==3456366== 
==3456366== HEAP SUMMARY:
==3456366==     in use at exit: 0 bytes in 0 blocks
==3456366==   total heap usage: 354 allocs, 354 frees, 53,880 bytes allocated
==3456366== 
==3456366== All heap blocks were freed -- no leaks are possible
==3456366== 
==3456366== For lists of detected and suppressed errors, rerun with: -s
==3456366== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
