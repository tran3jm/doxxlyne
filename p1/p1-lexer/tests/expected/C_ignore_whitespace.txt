SYM   [line 001]  !
SYM   [line 002]  ?
SYM   [line 003]  -
SYM   [line 004]  ,